requirements/base.txt

```diff
@@ -2,7 +2,6 @@
 elasticsearch==2.1.0
 Django==1.9.8
 djangorestframework==3.4.0
-psycopg2==2.6.2
```

Comment:
Why you delete psycopg2 from base requirements? will postgre still  be working on local machine ?